docs/modules/ROOT/pages/contributing/local-development.adoc

```diff
@@ -24,14 +24,26 @@ For other cluster types you may check the specific documentation. As soon as you
 make images
 ----
 
-This command will build and publish your actual Camel K development version to the container registry. At this stage installing the Camel K development version will be as easy as a typical installation:
+This command will build and publish your actual Camel K development version to the container registry. At this stage installing the Camel K development version will be as easy as a typical installation. We are supporting a series of makefile targets to let you easily install the operator on your local cluster in one single line. These targets are though to cover plain Kubernetes and Openshift cluster in either global (default in `camel-k` namespace) and namespaced (default in `default` namespace) installation:
 
 [source]
 ----
-./kamel install
+make install-k8s-global (default in camel-k namespace)
+make install-k8s-ns (default in default namespace)
+make install-openshift-global (default in camel-k namespace)
+make install-openshift-ns (default in default namespace)
 ----
 
-You can provide any customization required such as `--log-level`, `--operator-id`, etcetera. Mind that you need to use `./kamel` in order to use the development binary you've just built, as tipically, `kamel` is the production CLI you have installed in your binary path.
+Those targets may use two variables, `NAMESPACE` and `REGISTRY` in order to let you specify the namespace where to install the operator and the container registry to use. For instance:
+
+[source]
+----
+NAMESPACE=test REGISTRY=1.2.3.4 make install
```

Comment:
There is no `install` target, may be this is `install-operator` ?

Reply:
Good catch. It's a leftover from previous work. It is `install-k8s-global` or any other install target.